extensions/guacamole-auth-restrict/src/main/resources/templates/hostRestrictionField.html

```diff
@@ -0,0 +1,23 @@
+<div class="hostField">
+    <table class="restrictionList" ng-show="restrictions !== null && restrictions.length > 0">
+        <tr>
+            <th>Host</th>
+            <th><img src="images/x-red.svg" class="restrictionListHeader" alt="X"></th>
```

Comment:
What is the purpose of the red X's in the table headers?

Reply:
Just to denote that it is a column that contains the X to remove the entry from the table. I suppose that should be obvious based on the X, there, and I can probably remove the header.

Reply:
Removed via rebase.